### PR TITLE
i18n(zh-CN):Update 1.mdx in chinese

### PR DESCRIPTION
**Pull request by hackers267**
#### What kind of changes does this PR include?

- Translated content

#### Description

- What does this PR change? change the sentence '将你的网站的更新版本部署并部署到线上' to '将你网站的更新版本部署到网上', become the old sentence is not suit for Chinese.

### CODE REVIEW

File: src/content/docs/zh-cn/tutorial/2-pages/1.mdx [hunk 1]
```diff
@@ -15,7 +15,7 @@ import PreCheck from '~/components/tutorial/PreCheck.astro';
 <PreCheck>
   - 在你的网站上创建两个新页面：关于和博客
   - 为你的页面添加导航链接
-  - 将你的网站的更新版本部署并部署到线上
+  - 将你网站的更新版本部署到网上
```
Comment (100gle) [inline src/content/docs/zh-cn/tutorial/2-pages/1.mdx:18]:
```suggestion
  - 将你更新后的网站部署上线
```

**Review by 100gle — CHANGES_REQUESTED** [review]
我觉得这样改动之后读起来会更加通顺且简洁一些 :)

File: src/content/docs/zh-cn/tutorial/2-pages/1.mdx (hunk 1 above)
Comment (hackers267) [reply src/content/docs/zh-cn/tutorial/2-pages/1.mdx:18]: 我也赞成这样修改

Comment (100gle) [reply src/content/docs/zh-cn/tutorial/2-pages/1.mdx:18]: 你可以 Commit Suggestions，这样我就可以直接接着 Review 了

**Review by 100gle — APPROVED** [review]
LGTM!

**Comment by liruifengv** [comment]
Awesome! Nice change! Thank you @hackers267

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for this @hackers267 🙌

**Pull request merged**